### DISCUSSION TITLE
How do I get the green bean weight variable from Roast Properties

### DISCUSSION THREAD

**Discussion by Robjones6146** [artisan-roaster-scope/artisan#1727] [Q&A]
![weightVar](https://github.com/user-attachments/assets/1a135b20-18b9-4ca7-8dc2-f2ace5e0afb1)
Is there a way to get the Green bean weight from Roast Properties so I can use it in a calculation to automatically move my slider?

**Comment by MAKOMO** [comment]
Of course. The symbolic variable is named `WEIGHTin`. All variables defined are listed in the Help text accessed via the Help button in the extra devices dialog.

**Comment by Robjones6146** [comment]
Perfect. Thank you so much.

A year ago you shared this formula with me, what does the v stand for?
(100-v)
SLIDER: out(0,{},687340:4); out(0,100-v,687340:3)

**Reply by MAKOMO** [reply]
I cannot remember this (about 10k discussion items later) without the context. Sorry.

**Comment by Robjones6146** [comment]
I understand. I think it is the slider value. (v)

https://github.com/artisan-roaster-scope/artisan/issues/1230
